docs/Hardening.md

```diff
@@ -0,0 +1,197 @@
+# Linux Hardening while using OMS Log Collector
```

Comment:
think the "while ..." can be omitted

Reply:
Fixed in this commit:
https://github.com/microsoft/OMS-Agent-for-Linux/pull/1269/commits/b6a96c6bef0fd670a3c6676c17026d5228135af5#diff-ff8800189743d6567bcda8a83de5b22c943b28c51e4488cce0ea2b185716cc37R1

Reply:
I wasn't clear—why is this scoped to log collector? Isn't this hardening for OMS agent itself?

Reply:
...I'm an idiot, yes it is, sorry, I'll change it to OMS Agent

Reply:
Fixed in this commit: https://github.com/microsoft/OMS-Agent-for-Linux/pull/1269/commits/3d7de51694edac07a8c333f03a33464faa4347fc#
I wanted to keep the "OMS Agent" specification just to make sure it was clear that this has to do with hardening compatibility with OMS (if the fact that it was in our repo wasn't clear enough)